syntax = "proto3";

package stream.pb;

service StreamTest {
  rpc Test(stream TestRequest) returns (stream TestResponse) {}
}

message TestRequest {
  string name = 1;
  string text = 2;
}

message TestResponse {
  string name = 1;
  string text = 2;
}

